src/components/search-list/search-list.ts

```diff
@@ -84,8 +94,14 @@ export class SearchListComponent {
     this.navCtrl.push(ResultsPage);
   }
   
+  //Go to the TestsLists Page if user clicks on the option
   goToTestsPage(){
     this.navCtrl.push(TestListsPage);
   }
+  
+  //When user selects a new time range, update their tests view
+  changeTimeView(){
```

Comment:
This function here and where it is called on the html, I do not believe are needed.

Reply:
K, updates made